library/private/interactor_impl.h

```diff
@@ -84,6 +84,8 @@ class interactor_impl : public interactor
   void UpdateRendererAfterInteraction();
 
 private:
+  void AnimateCameraTransition(const std::function<void(camera&, double)>& update_camera);
```

Comment:
Add some doc for your fellow F3D developers :)

Reply:
wait, this method is in `interactor_impl::internals`, so the declaration is not needed anyway